src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -97,16 +97,24 @@
 
     @Column(name = "max_complaints")
     @JsonView(QuizView.Before.class)
-    private Integer maxComplaints;
+    private int maxComplaints;
```

Comment:
Is there any particular reason for this change? There was probably a reason why this was nullable.
You should probably check with Stephan and make sure that there are no entries in the production database where these two attributes are null.

Reply:
Also, it would make sense to also change this in the database. (Set the columns to NOT NULL)

Reply:
There was no concrete reason for this, I just wondered why Integer was used instead of int, even though I couldn't see why this was necessary :thinking: 

But you are right, I think we can leave it in this PR as it was before, and then in a new PR consider whether the data type choice is useful or necessary and check with Stephan.